extensions/standard-processors/controllers/UnorderedMapKeyValueStoreService.h

```diff
@@ -0,0 +1,75 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __UNORDERED_MAP_KEY_VALUE_STORE_SERVICE_H__
```

Comment:
I think this should be called "volatileKeyValueStoreService" to maintain consistency with our repos.

Reply:
I can very well imagine having multiple volatile implementations of the KeyValueStoreService in the future, with different properties than this, and I didn't want to designate this *the* VolatileKeyValueStoreService.

Reply:
Okay, I'm convinced.